### ISSUE TITLE
Enable git LFS

### ISSUE THREAD

**Issue by D4N** [Exiv2/exiv2#896]
We should enable git LFS or create an additional repository with git LFS enabled to store large test files, like the one mentioned in #810.

Any objections, suggestions, comments?

**Comment by clanmills** [comment]
I agree with this and would like to see this dealt with in v1.00 in #1550.

**Comment by 1div0** [comment]
Good idea @D4N , thanks for proposing.
